services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -656,29 +672,35 @@ class AudioColumn(MediaColumn):
     transform_column = FloatColumn
 
     @staticmethod
-    def get_duration(example: dict[str, Any]) -> float:
+    def get_duration(example: Optional[Union[bytes, dict[str, Any]]], is_struct: bool = True) -> Optional[float]:
```

Comment:
and therefore you don't need `is_struct `
```suggestion
    def get_duration(example: Optional[Union[bytes, dict[str, Any]]]) -> Optional[float]:
```